tests/test_webhooks.py

```diff
@@ -0,0 +1,42 @@
+from unittest import mock
+
+import pytest
+from rest_framework import serializers
+
+from drf_spectacular.generators import SchemaGenerator
+from drf_spectacular.utils import OpenApiResponse, OpenApiWebhook, extend_schema
+from tests import assert_schema
+
+
+class EventSerializer(serializers.Serializer):
+    id = serializers.CharField(read_only=True)
+    change = serializers.CharField()
+    external_id = serializers.CharField(write_only=True)
+
+
+urlpatterns = []  # type: ignore
+
+subscription_event = OpenApiWebhook(
+    name='SubscriptionEvent',
+    decorator=extend_schema(
+        summary="some summary",
+        description='pushes events to a webhook url as "application/x-www-form-urlencoded"',
+        request={
+            'application/x-www-form-urlencoded': EventSerializer,
+        },
+        responses={
+            200: OpenApiResponse(description='event was successfully received'),
+            '4XX': OpenApiResponse(description='event will be retried shortly'),
+        },
```

Comment:
Schema generation fails with the following message when responses is not set:

```
Error: unable to guess serializer. This is graceful fallback handling for APIViews. Consider using GenericAPIView as view base class, if view is under your control. Either way you may want to add a serializer_class (or method). Ignoring view for now.
```

I have not investigated how to solve it yet.

Reply:
i'll have a look.